generic/tests/build.py

```diff
@@ -30,5 +28,5 @@ def run(test, params, env):
             minor_failure_reasons.append(reason)
 
     if minor_failure:
-        raise error.TestWarn("Minor (worked around) failures during build "
-                             "test: %s" % ", ".join(minor_failure_reasons))
+        test.cancel("Minor (worked around) failures during build "
+                    "test: %s" % ", ".join(minor_failure_reasons))
```

Comment:
Should be `test.error` or `logging.warning`

Reply:
updated, thanks